apps/condo/domains/onboarding/schema/SyncTourStepsService.js

```diff
@@ -2,11 +2,18 @@
  * Generated by `createservice onboarding.SyncTourStepsService '--type=mutations'`
  */
 
+const isEmpty = require('lodash/isEmpty')
+
 const { GQLError } = require('@open-condo/keystone/errors')
 const { GQLCustomSchema, find, itemsQuery } = require('@open-condo/keystone/schema')
 
 const access = require('@condo/domains/onboarding/access/SyncTourStepsService')
 const { SYNC_TOUR_STEPS_ERRORS } = require('@condo/domains/onboarding/constants/errors')
+const {
+    CREATE_PROPERTY_STEP_TYPE, CREATE_PROPERTY_MAP_STEP_TYPE, CREATE_TICKET_STEP_TYPE,
```

Comment:
(optional) It's usual 1 per line, or grouped by logic per line, or all in 1 line. A bit strange. But just a tip for future, not requirement